uint8[3] Docking